src/compiler.c

```diff
@@ -5983,6 +6004,7 @@ static Int InitKernel (
     CompExprFuncs[ T_INT_EXPR        ] = CompIntExpr;
     CompExprFuncs[ T_TRUE_EXPR       ] = CompTrueExpr;
     CompExprFuncs[ T_FALSE_EXPR      ] = CompFalseExpr;
+    CompExprFuncs[ T_FALSE_EXPR      ] = CompTildeExpr;
```

Comment:
Hmm, shouldn't `T_FALSE_EXPR` be `T_TILDE_EXPR` here?

I guess we need to improve coverage of `compiler.c` -- say by crafting input files which trigger as much of the code in it as possible, then compiling them as part of the travis tests...